Packs/BigFix/Integrations/BigFix/BigFix.py

```diff
@@ -791,7 +791,7 @@ def test():
     elif demisto.command():
         query_command()
 
-except Exception, e:
+except Exception as e:
```

Comment:
This is python 2 integration, not python 3

Reply:
This syntax works in BOTH.